_episodes/21-carpentries.md

```diff
@@ -30,27 +30,27 @@ and on the final steps toward certification.
 
 ## History
 
-[Software Carpentry][swc] was co-founded in 1998 by Brent Gorda and
+[Software Carpentry]({{ site.swc_site }} ) was co-founded in 1998 by Brent Gorda and
 Greg Wilson, who identified a need for best practices training in
 research computing.  After several iterations, the current model of
 two-day workshops with a standard curriculum emerged in 2010-11.
 After intermediate support from various organizations, it became an
 independent non-profit organization called the [Software Carpentry
-Foundation][scf] (SCF) in 2015.  The SCF is now responsible for all
+Foundation]({{ site.swc_site }}/scf/) (SCF) in 2015.  The SCF is now responsible for all
```

Comment:
I believe the line-break in the middle of the square bracket will break this link, no?